client.lua

```diff
@@ -475,7 +475,7 @@ CreateThread(function()
 		Wait(50)
 		local ped = cache.ped
 		if isPedDrivingAVehicle() then
-			vehicle = GetVehiclePedIsIn(ped, false)
+			vehicle = GetVehiclePedIsIn(ped, true)
```

Comment:
Why this change?

Reply:
Error happens on exiting a vehicle:

[ 10295266] [b2802_GTAProce] MainThrd/ no entity for vehicle 0 in GET_VEHICLE_HANDLING_FLOAT
[ 10295266] [b2802_GTAProce] MainThrd/ no script guid for vehicle 0 in SET_VEHICLE_HANDLING_FIELD

It happens after you get in and out, it looks like that variable is getting replaced by 0 when GetVehiclePedIsIn is false instead of giving the last vehicle in, so when the condition is not met below, it breaks the handling. You can test and see that if you exit a vehicle multiple times now it won't give this error any longer and the breaks will continue to work, I am guessing also the collision and damage multiplier were also being affected by this.

Reply:
```suggestion
			vehicle = cache.vehicle
```